modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDynamicParameterTest.java

```diff
@@ -159,8 +160,11 @@ public void testWithDifferentParametersTypes() {
      */
     @Test
     public void testWithDifferentParametersTypesMismatch() {
-        assertThrows(IgniteException.class, () -> assertQuery("SELECT COALESCE(12.2, ?)").withParams("b").check());
-        assertThrows(IgniteException.class, () -> assertQuery("SELECT COALESCE(?, ?)").withParams(12.2, "b").check());
+        SqlException t = assertThrows(SqlException.class, () -> assertQuery("SELECT COALESCE(12.2, ?)").withParams("b").check());
```

Comment:
there is org.apache.ignite.internal.sql.engine.util.SqlTestUtils#assertThrowsSqlException

Reply:
Thanks, I've fixed all the places in the PR where applicable.